.github/workflows/ci.yml

```diff
@@ -94,3 +94,13 @@ jobs:
       - uses: actions/checkout@v2
       - uses: dtolnay/rust-toolchain@clippy
       - run: cargo clippy --workspace --tests -- -Dclippy::all
+
+    clang-tidy:
+      name: Clang Tidy
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+        - name: Install clang-tidy
+          run: sudo apt install clang-tidy-10
+        - name: Run clang-tidy
+          run: clang-tidy-10 src/cxx.cc --header-filter='cxx\.h' -- -Iinclude -xc++ -std=c++11
```

Comment:
Minor nitpicks:
- don't think the `-Iinclude` is needed due to the way `cxx.cc` includes `cxx.h`. At the same time, it does not do any harm.
- why specifying C++ with `-xc++`? The `.cc` extension should be sufficient enough for clang to detect it's a C++ file or is it not?